src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -599,6 +599,11 @@ public ActivityProfileState isActive( final AlignmentContext context, final Refe
         }
 
         final SortedSet<VariantContext> allVariationEvents = untrimmedAssemblyResult.getVariationEvents(hcArgs.maxMnpDistance);
+        for (final VariantContext given : givenAlleles) {
+            if (allVariationEvents.stream().noneMatch(vc -> vc.getStart() == given.getStart() && vc.getAlternateAllele(0).basesMatch(given.getAlternateAllele(0)))) {
```

Comment:
These will always be trimmed right?  (And always biallelic?) Because they're each for a single haplotype?  If you agree, then let's add a comment for posterity.